docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+
+services:
+  regulome:
+    image: regulome
```

Comment:
I think `image` key is meant to be used for images that are available in some online registry. For local stuff you can replace this with `build: .`.